core/server/master/src/main/java/alluxio/master/AlluxioSimpleMasterProcess.java

```diff
@@ -159,113 +108,30 @@ public void start() throws Exception {
    */
   @Override
   public void stop() throws Exception {
-    stopRejectingServers();
-    if (isGrpcServing()) {
-      stopServing();
-    }
+    mServices.forEach(SimpleService::stop);
     mJournalSystem.stop();
-    stopMaster();
+    stopMasters();
     mLeaderSelector.stop();
   }
 
-  protected void startMaster(boolean isLeader) {
+  protected void startMasters(boolean isLeader) {
```

Comment:
i'm a bit paranoid that people may confuse `masters` here with the master processes in HA. what about `startMasterComponents`? Any suggestions?